scripts/th/th-doctor.sh

```diff
@@ -142,38 +144,56 @@ print_framed_text() {
     }
 
     show_more() {
-        # Test Harness Network / DUT
+        # Test Harness DUT Configuration
         th_net_dut_output=$("$TH_DEV_SCRIPTS_DIR/_th-net-dut.sh")
-        print_framed_text "$th_net_dut_output" "Test Harness Network / DUT"
+        print_framed_text "$th_net_dut_output" "Test Harness DUT Configuration"
+    }
 
-        # Docker Containers
-        docker_images_output=$("$TH_DEV_SCRIPTS_DIR/_th-docker-containers.sh")
-        print_framed_text "$docker_images_output" "Docker Containers"        
+    show_docker_logs() {
+        # Test Harness Docker Container Logs
+        th_net_dut_output=$("$TH_DEV_SCRIPTS_DIR/_th-docker-logs.sh")
+        # print_framed_text "$th_net_dut_output" "Test Harness Docker Container Logs"
```

Comment:
Was this commented out line left here intentionally or should it be removed?

Reply:
Removed.